### ISSUE TITLE
Training Loss Not Improving

### ISSUE THREAD

**Issue by sherrychen127** [BestJuly/IIC#6]
Hi, I'm trying to reproduce your results by training on ucf101. I noticed that my loss is not improving at all. I trained with a batch size of 20, and the loss is stuck at around 15.2 for every iteration, and does not decrease. Just wondering what's the configuration that you used to train the model?

**Comment by BestJuly** [comment]
Hi, @sherrychen127. Thank you for your interest.

The configurations are set as default. Have you ever tried our default settings? 

For default settings, we have fixed all random seed so that you should achieve exactly the similar results as ours if you use networks such as C3D, R21D. For R3D, the results in our paper (based on R3D) were trained without fixing all random seed, but they should be very similar.
Another thing is that if you use a different cuda version, there might be some difference in results. (Or the same, I am not sure and have not tried different experimental environment)

If there is still problems with your training process, please pose your training settings here and I can try with your settings to check whether something wired happens in my experimental environment.
